google/cloud/forseti/notifier/notifiers/inventory_summary.py

```diff
@@ -154,13 +158,35 @@ def transform_to_template(data):
         Returns:
             list: Sorted data as a list of dicts.
                 Example: [{resource_type, count}, {}, {}, ...]
-
         """
         template_data = []
         for key, value in data.iteritems():
             template_data.append(dict(resource_type=key, count=value))
         return sorted(template_data, key=lambda k: k['resource_type'])
 
+    @staticmethod
+    def _get_gsuite_dwd_status(summary_data):
+        """Get the status of whether G Suite DwD is enabled or not.
+
+        Args:
+            summary_data (list): Summary of inventory data as a list of dicts.
+                Example: [{resource_type, count}, {}, {}, ...]
+
+        Returns:
+            str: disabled or enabled.
+        """
+        gsuite_types = set(['gsuite_group', 'gsuite_user'])
+        summary_data_keys = set()
+        if summary_data is None:
+            return 'disabled'
+
+        for resource in summary_data:
+            summary_data_keys.add(resource['resource_type'])
+
+        if gsuite_types.issubset(summary_data_keys):
```

Comment:
Is it possible ( I think it is ) to have gsuite users but not groups? This will return false if you only have users but it is enabled because the users are not using groups.

Reply:
@joecheuk I have taken out gsuite_group from the set of fields to look for - please take a look and let me know if there is anything additional I can change.